book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  # has many rentals
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.book = self
+  end
```

Comment:
- [ ] Great job adding a method for adding rentals 👏🏻 However, the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. Kindly address this 👍🏻 

- You can make reference to the `add_rental` method of the `class Person`